modules/openy_features/openy_prgf/modules/openy_prgf_1c/openy_prgf_1c.info.yml

```diff
@@ -3,6 +3,7 @@ description: 'Implements a paragraph with 1 column.'
 type: module
 core: 8.x
 package: OpenY
+version: '8.x-1.0'
 dependencies:
```

Comment:
This paragraph also has following modules in the list of dependencies
  - openy_block_flexible_content
  - openy_block_custom_simple

Do we really need them here? Does it make sense to decouple in the current PR?

Reply:
Fixed for `openy_block_flexible_content`, but we need at least 1 block in target columns, so `openy_block_custom_simple` will be required by openy profile and can't be decoupled